content/applications/marketing/marketing_automation/getting_started/first_campaign.rst

```diff
@@ -1,59 +1,108 @@
 
-.. image:: first_campaign/marketing-template-sample.png
+.. image:: first_campaign/blank-marketing-campaign-form.png
    :align: center
-   :alt: A dashboard showing the creation of a new marketing automation campaign in Odoo.
-
-- :guilabel:`Templates`: represents the number of pre-configured mail templates being used in this
-  particular campaign. (Templates can always be created on-the-fly as well).
-- :guilabel:`SMS`: represents the number of personalized SMS messages connected to this campaign.
-- :guilabel:`Clicks`: represents the number of times attached links have been clicked by recipients
-  of this campaign.
-- :guilabel:`Participants`: represents the number of contacts that have directly participated in
-  this campaign.
-- :guilabel:`Target`: this field is a drop-down menu to choose which model is targeted by this
-  campaign (i.e., by Contacts, Sales Order, Lead/Opportunity, etc.).
-
-Campaign filters
-================
-
-To add a filter to the target audience, click :guilabel:`Add Filter`, and a node field will
-appear. In the node field, a custom equation can be configured for Odoo to use when filtering who
-to include (and exclude) in this specific marketing campaign.
+   :alt: A blank marketing automation campaign form in Odoo Marketing Automation application.
 
-.. image:: first_campaign/filter-node.png
-   :align: center
-   :alt: A filter node in Odoo Marketing Automation.
+After entering a name for the marketing campaign, configure the target audience in the remaining
+fields.
+
+In the :guilabel:`Target` field, use the drop-down menu to choose which model the target audience
+filters should be based on (e.g. :guilabel:`Contact`, :guilabel:`Lead/Opportunity`,
+:guilabel:`Sales Order`, etc.).
+
+Select :guilabel:`Search More...` from the drop-down menu to reveal a :guilabel:`Search: Target`
+pop-up window containing all 157 target options.
+
+Once a :guilabel:`Target` is selected, there's a :guilabel:`Unicity based on` field. This field is
+used to avoid duplicates based on the model chosen in the :guilabel:`Target` field.
+
+.. example::
+   If :guilabel:`Customers` is chosen as the :guilabel:`Target`, select :guilabel:`Email` in the
+   :guilabel:`Unicity based on` field so Odoo doesn't process any records that have the same Email
+   address.
+
+Select :guilabel:`Search More...` from the :guilabel:`Unicity based on` drop-down menu to reveal all
+129 options in a pop-up window.
 
-When the first field of the node is clicked, a nested drop-down menu of options appears on the
-screen where specific criteria is chosen based on needs of the campaign. The remaining fields on
-the node further define the criteria which determines which records to include (or exclude) in the
-execution of the campaign.
+Lastly on the campaign detail form is the :guilabel:`Filter` field. This is where a more specific
+audience filter can be configured to add specificity to the recipients of the marketing automation
+campaign.
```

Comment:
What are a few examples of the kinds of extra-specific configurations we can add here?

Reply:
Well, this is a bit tricky. Since there are literally hundreds of random configurations, I don't want to list a few, and the reader wrongfully assumes they can't specify something they want to. So, for the sake of avoiding confusion, I'm going to leave this open-ended, but I fully understand the suggestion. Thanks for pointing this out.